java/code/src/com/redhat/rhn/domain/org/OrgFactory.java

```diff
@@ -336,23 +336,17 @@ public static Org getSatelliteOrg() {
      * @param channelFamily Channel family to search for.
      * @return List of orgs.
      */
-    public static List<Org> lookupOrgsUsingChannelFamily(
-            ChannelFamily channelFamily) {
-        Map<String, Object> params = new HashMap<>();
-        params.put("cf", channelFamily);
-        return singleton.listObjectsByNamedQuery(
-                "Org.findOrgsWithSystemsInChannelFamily", params);
+    public static List<Org> lookupOrgsUsingChannelFamily(ChannelFamily channelFamily) {
+        Map<String, Object> params = Map.of("cf", channelFamily);
```

Comment:
Why didn't you put this map.of directly in the call, like in the other?

Reply:
I left some of those outside the call if they needed to be on a second line because of the 120 chars rule. There isn't much difference between the two solutions